org.freedesktop.tuhi.json

```diff
@@ -77,8 +77,9 @@
             "buildsystem": "simple",
             "sources": [
                 {
-                    "type": "git",
-                    "url": "git://anongit.freedesktop.org/xdg/pyxdg"
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/47/6e/311d5f22e2b76381719b5d0c6e9dc39cd33999adae67db71d7279a6d70f4/pyxdg-0.26.tar.gz#md5=db1c2af8300ca64ce3955b3cf2490c92",
+                    "sha512": "f5306e6e15af07df2599017500fc8ad83e722e5d5c6e4fda014aab1d77df92a3c3199a5be7a889faaecab72861e9910be9d80142d29856eb7a11f6ab9a923bd2"
```

Comment:
I notice pygobject is still on git, was this intentional? For that at least, using the archive from gitlab seems like the better option: https://gitlab.gnome.org/GNOME/pygobject/repository/3.27.5/archive.zip

Reply:
I sneaky pushed 9e85b4d2485e9630610c1a887b0a04bcf62e634f#diff-87adc521582e778cea8494987805dba5 which incudes the ftp host for pygobject